test/start-kubernetes.sh

```diff
@@ -78,7 +78,7 @@ case ${TEST_DISTRO} in
         ;;
 esac
 
-SSH_TIMEOUT=60
```

Comment:
So the cluster now comes up slower than before? That's worrisome, but not much that we can do about it. :man_shrugging:

Reply:
To my surprise, It was not observed in CI but on my dev machine, the failure was quite frequent.